source/features/bugs-tab.tsx

```diff
@@ -49,7 +57,7 @@ const bugs = new CachedFunction('bugs', {
 
 async function getSearchQueryBugLabel(): Promise<string> {
 	const {label} = await bugs.getCached() ?? {};
-	return 'label:' + SearchQuery.escapeValue(label ?? 'bug');
+	return `(label:${SearchQuery.escapeValue(label ?? 'bug')} OR type:Bug)`;
```

Comment:
My main concern with this is that selecting the `bug` label will no longer activate the "bugs" tab

<img width="497" alt="Screenshot" src="https://github.com/user-attachments/assets/c7cbaaf0-4aab-4c38-b2da-0cf98cc1239c" />

But that's ok because now that's not enough to show all bugs in the repo. Thanks GitHub.